pangeo-notebook/conda-linux-64.lock

```diff
@@ -492,8 +483,7 @@ https://conda.anaconda.org/conda-forge/noarch/nbstripout-0.5.0-pyhd8ed1ab_0.tar.
 https://conda.anaconda.org/conda-forge/noarch/nc-time-axis-1.4.1-pyhd8ed1ab_0.tar.bz2#281b58948bf60a2582de9e548bcc5369
 https://conda.anaconda.org/conda-forge/noarch/odc-geo-0.1.3-pyhd8ed1ab_0.tar.bz2#7ed4930acc7a24889df8a0c4bd54979c
 https://conda.anaconda.org/conda-forge/noarch/prompt-toolkit-3.0.29-pyha770c72_0.tar.bz2#9e720b57b22ef3032b4fb081697819dd
-https://conda.anaconda.org/conda-forge/linux-64/pyarrow-8.0.0-py39h42d110c_1_cpu.tar.bz2#9a6fd682c08e88c5b0e18ba84916b2b8
-https://conda.anaconda.org/conda-forge/linux-64/pyqt-5.12.3-py39hf3d152e_8.tar.bz2#466425e3ee3b190e06b8a5a7098421aa
+https://conda.anaconda.org/conda-forge/linux-64/pyarrow-6.0.1-py39h42d110c_19_cpu.tar.bz2#ce21bb03b644c34f5626a872fa393e2d
```

Comment:
note pyarrow drop from 8.0 -> 6.0, also pyqt is finally gone b/c of new `parcels` package!